src/item.cpp

```diff
@@ -1699,6 +1703,10 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
             }
         }
 
+        for( const auto &e : faults ) {
+            info.emplace_back( "DESCRIPTION", string_format( _( "* <bad>Faulty %s</bad>.  %s" ), e.obj().name().c_str(), e.obj().description().c_str() ) );
```

Comment:
Comment for translators (using ```//~```) about content of the formatted string will be useful here, I guess.